R/LD.R

```diff
@@ -174,7 +174,7 @@ process_LD_matrix <- function(LD_file_path, bim_file_path) {
     # Process variant names from file paths
     LD_variants <- bim_file_name %>%
               read.table(.) %>%
-              setNames(c("chrom","variants","GD","pos","A2","A1"))%>%
+              setNames(c("chrom","variants","GD","pos","A1","A2"))%>%
```

Comment:
are you sure this line change is correct? A2 should be position 3, A1 position 4?

Reply:
When Tosin make the bim file I think it is also done by this order, see his code on this page:

https://cumc.github.io/xqtl-pipeline/code/reference_data/ld_reference_generation.html

the part that bim file are written

Reply:
I checked it and also searched in plink documentation:
<img width="649" alt="image" src="https://github.com/cumc/pecotmr/assets/113001086/facd83a1-0e68-4d12-9ffd-4637d763b97d">

so I think my update here is correct

Reply:
A1 is alt (effect allele) and A2 is ref (other allele). So A1 in front of A2. In Tosin's code it's correct: alt in front of ref, which means A1 in front of A2